hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -6333,6 +6333,20 @@
     </description>
   </property>
 
+  <property>
+    <name>dfs.journalnode.maintenance.nodes</name>
+    <value></value>
+    <description>
+      In the case of one out of three journal nodes being down, theoretically the service can still
+      continue. However, in reality, the downed node may not recover quickly. If the Namenode needs
+      to be restarted, it will try the downed journal node through the lengthy RPC retry mechanism,
+      resulting in a long initialization time for the Namenode to provide services. By adding the
+      downed journal node to the maintenance nodes, the initialization time of the Namenode in such
+      scenarios can be accelerated.
```

Comment:
nit

->

 In the case that one out of three journal nodes being down, theoretically HDFS can still
      function. However, in reality, the unavailable journal node may not recover quickly. During this period, when we need to restart an Namenode, the Namenode will try to connect to the unavailable journal node through the lengthy RPC retry mechanism,
      resulting in a long initialization time for the Namenode. By adding these
      unavailable journal nodes to the maintenance nodes, we will skip these unavailable journal nodes during Namenode initialization and thus reduce namenode startup time.

1-node example values: <>
2-node example values: <>